course_discovery/settings/devstack.py

```diff
@@ -22,6 +22,12 @@
 
 JWT_AUTH.update({
     # Must match public signing key used in LMS.
+    'JWT_ISSUERS':
+        [{
+            "AUDIENCE": "lms-key",
+            "ISSUER": "http://local.philanthropyu.org:18000/oauth2", 
+            "SECRET_KEY": "lms-secret"
+        }],
```

Comment:
move this above this comment `# Must match public signing key used in LMS.` and properly format

Reply:
Done